sale_automatic_workflow/models/automatic_workflow_job.py

```diff
@@ -46,12 +49,12 @@ def _do_send_order_confirmation_mail(self, sale):
         sale._send_order_confirmation_mail()
 
     @api.model
-    def _validate_sale_orders(self, order_filter):
+    def _validate_sale_orders(self, order_filter, auto_commit=False, limit=None):
```

Comment:
To avoid passing the auto_commit parameter at each function, shouldn't it be better to get the auto_commit value from the sale order workflow directly?

So in the iteration loop on sales:

sale.workflow_process_id.auto_commit